renderdoc/common/common.h

```diff
@@ -294,6 +296,18 @@ inline T AlignUpPtr(T x, A a)
   return (T)AlignUp<uintptr_t>((uintptr_t)x, (uintptr_t)a);
 }
 
+template <typename T>
+std::enable_if_t<std::is_integral<T>::value, T> AlignToMultiple(T value, T multiple)
```

Comment:
Please don't use this template metaprogramming stuff especially in a global header, it is unnecessarily complicated and adds compile bloat from the header and template evaluation for little gain.

Assuming you want to prevent someone accidentally using this on floats and getting the wrong result add an `| 0` onto the end of the expression and you'll get a compile error for floats and a no-op for ints. Otherwise this can be a simple template similar to `AlignUp`.